packages/ag-charts-community/src/options/series/cartesian/bulletOptions.ts

```diff
@@ -25,8 +25,13 @@ export interface AgBulletScaleOptions {
 }
 
 export interface AgBulletSeriesThemeableOptions extends AgBulletSeriesStyle, AgBaseSeriesThemeableOptions {
+    /* Width of the bar relative to the width/height of the series area. */
+    widthRatio?: number;
     /** Styling options for the target node. */
-    target?: AgBulletSeriesStyle;
+    target?: AgBulletSeriesStyle & {
```

Comment:
@olegat FYI, we should avoid using type literals inside the options files because it breaks the documentation. We cannot display the type of this property because a type literal as no name.

Reply:
Okay noted. The names of these options are still TBD which is why I haven't bothered creating separate typename yet.

Reply:
I get it... but in future builds this will throw an error and fail the build.
Better to have a temp name than no name at all :)